src/components/mdx_wrapper.tsx

```diff
@@ -61,7 +63,7 @@ const MdxWrapper: FC<MdxWrapperProps> = ({Component}) => {
         return (
           <div className="container" id="mdx-container">
             {/* doc will be undefined for the very first render */}
```

Comment:
Could you change the comment to `/* metadata and data will be undefined ...`?

Reply:
done